### DISCUSSION TITLE
'Custom' dependabot with git submodules?

### DISCUSSION THREAD

**Discussion by Ismoh** [community/community#28251] [Apps, API and Webhooks]
Hey there,
I have a github repo and some dependencies by simply copy pasted lua files in for example /libs/foo.lua and /libs/bar.dll
foo.lua and bar.dll are files from github public repos. I simply downloaded the release and extracted those files. Copy pasted then in my repo.

Then some days later I saw dependabot, but didnt found anything regarding lua.
I know dependabot doesnt work with lua, but git submodules. I think this would be a workaround!

Does anyone already tried git submodules?
Because I really like and love dependabots dependencies list and also the update functionality and PR stuff.

If anyone has some resources, please let me know!

I didnt understand githubs dependabot docs right. So could use some additional resources.

Thanks a ton!

**Comment by zoranbosnjak** [comment]
Hi @Ismoh,
I have similar problem.
Have you found any solution from what you describe?

**Reply by Ismoh** [reply]
Hey, there isn't any solution yet. Someone is working on a PR on dependabot-core to be able to use dependabot with Lua. Sorry, that's all I can say. I am using LuaRocks for depensencies now, but setting this up, was a pain! And I have to check for updates manually.

**Reply by Ismoh** [reply]
> Hey, there isn't any solution yet. Someone is working on a PR on dependabot-core to be able to use dependabot with Lua. Sorry, that's all I can say. I am using LuaRocks for depensencies now, but setting this up, was a pain! And I have to check for updates manually. 

In addition: git submodules was even more painfull, because you will pull git content you don't need or want. It's kinda like a checkout. That's why I decided to use LuaRocks.
